frontend/pages/viewer-registration/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { FormEvent, useState } from 'react'
 import styles from './viewer-registration.module.css'
-import PasswordAid from './password-aid/password-aid'
+import PasswordAid from '../../shared-components/password-aid/password-aid'
```

Comment:
This was particularly kept out of shared components because it won't be used anywhere else in the app except on the registration page

Is there a particular reason we can't have children components in their parent's directory?

Reply:
Okay, hmm It's a pattern that I'm used to but that works to - in this case though wondering if we might be able to revisit the idea of having sub-directories in the shared-components directory? 
Seems like shared-components is going to get crowded quickly if it's really just all non-page components

Reply:
I think having one sub-directory level deep in the shared-components folder is more than reasonable.